### ISSUE TITLE
Row Parsing Fails

### ISSUE THREAD

**Issue by IgnitedCoder** [thephpleague/csv#197]
## Issue summary

_(Please explain in plain english your issue/feature request)_

### System informations

_(In case of a bug report Please complete the table below)_

| Information | Description |
|--------------|---------|
| League\Csv ^8.0 |  |
| PHP 7.0 |  |
| MACOSX |  |

## Standalone code, or other way to reproduce the problem

Sample file import

"MSN	Title	Initials	Surname	Address1	Address2	Address3	Address4	PostCode	PammsReq	CopyChangeFlag"
"16321280091780	MR                  	J	VAN DER WALT	17 Cambridge Avenue	Craighall Park	NULL	NULL	2196 	NULL	A63"
"16321280091960	MISS                	ZB        	DYUBERA                                 	335 Seyisi Street	Adelaide			5760 	NULL	B63"

_(Please complete the text below to help us fix the issue)_

### Expected result

$row parsed with field collection.

### Actual result

$row = array with 1 item representing the row. i.e. 1 string, not a collection of columns even though I've set the enclosure to ", note the file is tab seperated with double quote enclosure per row.

**Comment by nyamsprod** [comment]
@IgnitedCoder please provide an actual PHP code as I can not reproduce your issue and thus determine if it is a bug or a misusage of the library.

**Comment by IgnitedCoder** [comment]
Absolutely... I think it might just be user error, that would be good news. :)

$dataFile = $this->template->filePath . $this->template->fileName;
        $this->fileReader = Reader::createFromPath($dataFile);
        $this->fileReader->setDelimiter('    '); //Tab delimited character.
        $this->fileReader->setEnclosure('"');
        $this->fileReader->setEscape('\\');

Note this works when I remove the double quote enclosures..

**Comment by nyamsprod** [comment]
your file is not a real tab separated file if each row starts with a '"'. That's the issue.

**Comment by IgnitedCoder** [comment]
yeah I guess, each row is encosed in double quotes and is tab seperated. Any ideas? I would very much like to continue using the CSV Lib.

**Comment by nyamsprod** [comment]
If no values inside the row uses the '"' you could use [the stream filter](http://csv.thephpleague.com/filtering/) to remove the '"' prior to reading
